scss/footer.scss

```diff
@@ -0,0 +1,22 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  color: $white;
+  display: flex;
+  flex-direction: column;
+  padding: 40px 0 30px;
+  text-align: center;
+
+  .footer-disc {
+    font-size: $text-big;
+    margin: 0 0 12px;
+    text-transform: uppercase;
+  }
+
+  .footer-copyright {
+    align-self: flex-end;
+    font-size: 11px;
+    font-weight: 300;
+    margin-right: 20px;
```

Comment:
why this prop?

Reply:
to add some space between the screen and the copyright

Reply:
Add a padding to the footer